packages/spor-theme-react-native/src/components/expandable.tsx

```diff
@@ -0,0 +1,53 @@
+import { shadows } from "@vygruppen/spor-theme-react-native/src/foundations/shadows";
+import { colors } from "@vygruppen/spor-theme-react-native/src/foundations/colors";
+import { borderRadii } from "@vygruppen/spor-theme-react-native/src/foundations/borderRadii";
+import { spacing } from "@vygruppen/spor-theme-react-native/src/foundations/spacing";
+
+export const expandableVariant = {
+    defaults: {
+        borderRadius: "sm",
+    },
+    text: {
```

Comment:
Trengs denne?

Reply:
Jeg lurer på om den trengs for å ikke klikke når man sender inn variant="text"… usikker tho.

Reply:
Ja det stemmer, så enten for vi vi har den tom, eller så får variant bli til en optional.